app.rb

```
@@ -0,0 +1,30 @@
+require 'sinatra'
+require 'json'
+require_relative 'lib/test_results_processor'
+
+post '/api/test_results' do
+
+  request_body = request.body.read.strip
```

rubocop

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)